Dockerfile

```diff
@@ -8,7 +8,16 @@ COPY ./secrets /
 
 WORKDIR /src/
 
-CMD ["node", "/src/index.js"]
-
 COPY ./src/ /src/
-RUN npm install --production
+RUN npm install
+RUN npm rebuild
+
+FROM node:12.14.0-alpine 
+
+WORKDIR /src/
```

Comment:
I think it worth changing directory name to `/app`

Reply:
Maybe... I'll let @bushjames chime in as I think there are a few repos standardized on `/src/`

Reply:
@kirgene what is the rationale for changing /src to /app?

Reply:
@bushjames I think it's considered as best practice (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)

Reply:
I don't think this really matters at the moment, plus it has other implications for the directory structure on other projects.

I propse we should leave it as is for now, and if you want @kirgene maybe we can take it up as a question of standardization across mojaloop repos containing Dockerfiles.

Reply:
Yes, it's actually just a suggestion, so we can leave it as-is for now.